testsuite/features/secondary/srv_change_task_schedule.feature

```diff
@@ -5,7 +5,7 @@
 Feature: Change the schedule of a task
 
   Scenario: Log in as admin user
-    Given I am authorized for the "Admin" section
+    Given I am authorized
```

Comment:
In theory, running it with just `Given I am authorized` it will use a new user name "srv_change_task_schedule", and that user will not have admin rights. So, in theory, the navigation to the admin section should fail.

In any case if you still want to use a regular use, change also the Scenario title accordily.